### PR TITLE
Update schema.json

### PR DESCRIPTION
**Pull request by Eric-revollo**
@GustavoRamos89  review

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+{*
```
Comment (GustavoRamos89) [inline schema.json:1]: Remove asterisk

File: schema.json [hunk 2]
```diff
@@ -17,4 +17,4 @@
     }, {
         "name": "Temperature",
         "type": "number"
-    }
+    }}
```
Comment (GustavoRamos89) [inline schema.json:20]: Remove extra brace

**Review by GustavoRamos89 — CHANGES_REQUESTED** [review]

File: schema.json (hunk 1 above)
Comment (Eric-revollo) [reply schema.json]: done

File: schema.json (hunk 2 above)
Comment (Eric-revollo) [reply schema.json]: done

**Pull request merged**